service/src/main/java/org/cbioportal/service/impl/SampleServiceImpl.java

```diff
@@ -36,6 +37,7 @@
     private CopyNumberSegmentRepository copyNumberSegmentRepository;
     
     @Override
+    @PostFilter("hasPermission(filterObject, 'read')")
```

Comment:
Can you check if `org.cbioportal.security.spring.CancerStudyPermissionEvaluator` `hasPermission` method supports `Sample` target object?